models/nft/ethereum/nft_ethereum_top_erc1155_holders.sql

```diff
@@ -0,0 +1,44 @@
+{{ config(
+       alias = 'top_erc1155_holders',
+       materialized='table',
+       post_hook='{{ expose_spells(\'["ethereum"]\',
+                                   "sector",
+                                   "nft",
+                                   \'["Henrystats"]\') }}'
+       )
+   }}
+
+ WITH 
+
+ erc1155_balances as (
+     SELECT 
+         wallet_address,
+         token_address as nft_contract_address,
+         COUNT(tokenId) as balance 
+     FROM 
+     {{ ref('balances_ethereum_erc1155_latest') }}
+     GROUP BY 1, 2
+ ), 
+
+ erc1155_ranked as (
+     SELECT 
+         wallet_address, 
+         nft_contract_address, 
+         balance, 
+         SUM(balance) OVER (PARTITION BY nft_contract_address) as total_supply, 
+         ROW_NUMBER() OVER (PARTITION BY nft_contract_address ORDER BY balance DESC) as rn 
+     FROM 
+     erc1155_balances
+     QUALIFY rn <= 50 
```

Comment:
i'd recommend ensuring this `qualify` works on dunesql too, for future. i'd also suggest standardizing the query format for both this and #3206

Reply:
I'll stamdardoze the format now.

Reply:
updated